tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+  "include": ["src", "types"],
+  "exclude": ["node_modules", "**/*.spec.ts"],
+  "compilerOptions": {
+    "module": "esnext",
+    "lib": ["esnext"],
+    "target": "es6",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "strict": true,
+    "rootDir": "./src",
+    "outDir": "./lib",
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "types": ["jest", "node"],
+    "skipLibCheck": true,
+    "typeRoots": ["./node_modules/@types", "./types"]
```

We don't have a `types` dir. This option can be removed.
```suggestion
    "skipLibCheck": true
```